crawler/file_processing.py

```diff
@@ -11,99 +11,58 @@
 from hashlib import md5
 from logging import INFO, WARN
 from pathlib import Path
-from typing import Any, Dict, Final, List, Optional, Set, Tuple, cast, Iterator
+from typing import Any, Dict, Final, Iterator, List, Optional, Set, Tuple, cast
 
 from bson.decimal128 import Decimal128
 from more_itertools import groupby_transform
 from pymongo.database import Database
 from pymongo.errors import BulkWriteError
 
-from crawler.constants import (
```

Comment:
Seems there are many large diffs where the only difference is in the order of imports, what are your setting for python in VSCode? Here are mine:
```
"python.autoComplete.addBrackets": true,
"python.formatting.provider": "black",
"python.languageServer": "Pylance",
"python.analysis.completeFunctionParens": true,
"python.analysis.typeCheckingMode": "basic",
"[python]": {
    "editor.codeActionsOnSave": {
      "source.organizeImports": true
    },
    "gitlens.codeLens.symbolScopes": ["!Module"],
    "editor.wordBasedSuggestions": false
  },
```